content/english/blog/2024-06-open-sauce.md

```diff
@@ -0,0 +1,172 @@
+
+A huge thanks and shout out to Noisebridge for getting our project running again!
+
+### Rebirth
+
+<img class="img-responsive" src="/images/blog/2024/open-sauce/broken-bot-3.jpg" title="The team works feverishly and diligently to rescue a viable encoder from the donor arm">
+<img class="img-responsive" src="/images/blog/2024/open-sauce/broken-bot-4.jpg" title="In a high-stakes operation, the new encoder is ready to be transplanted into our sick patient">
+
+With the donor encoder in, we were all super relieved to see our robot moving once again.  The arm proved to be no trouble
+up until the end of the event.
+
+### Tragedy (the second)
+
+As the event was drawing to a close, we were poised to end everything on a really high note: we'd sold out both days
+on our badge component kits, the robot was working again, and we got to see so many cool exhibitors showing off their
+cool creations.  A few creators had even stopped by our booth to play the ARM just after the official closing time.
+
+Now, some important background: as an artifact of being a barn in its past life, Cow Palace did not have the electrical infrastructure
+to power a bunch of weird robots, computers, and gadgets, necessitating the use of rented generators.  Knowing this, you
+might feel a little uneasy powering your precious 90's robot controller that you don't have a complete software backup for
+off that, but I took at least a little comfort in some industry knowledge about these things - these big generators are
+really just small, 3-phase power plants, frequently used to power expensive concert equpiment without incident - lighting,
+sound boards, speakers, extremely rare and expensive vintage synthesizers, the whole nine-yards.
+
+Unfortunately all of that
+stability comes to a deafening fizzle thanks to nature's tendency to invent idiots who suddenly believe they're qualified
+to go throwing switches on the generator's control panel. Just a few seconds into that final game, a brown out due to
+improper generator operation sent the arm crashing into the target controller on its back, and the address decoding
+circuit in the memory controller to send a logfile write, which normally goes to battery-backed RAM, destined instead
+to to the flash chips storing unreplaceable firmware and calibration values.
+
+<img class="img-responsive" src="/images/blog/2024/open-sauce/broken-bot-6.jpg" title="The robot controller's console now only boots to immediately kernel panic">
+
+
+Last year in the course of learning more about our controller, I'd attempted to take a backup of the internal flash
+on the controller, but after revisiting those backups I'm totally disheartened to learn that 1 of the 4 flash chip dumps
+is empty.
+
+If you know anyone with this era of robot arm controller, that could be instrumental in getting our project working
+again, please drop us an email here at info [at] pawprintprototyping.org.
```

Comment:
```suggestion
If you know anyone with this era of robot arm controller, it could be instrumental in getting our project working
again. Please drop us an email at info [at] pawprintprototyping.org.
```